src/admin/components/elements/Drawer/index.scss

```diff
@@ -34,9 +34,14 @@
     z-index: 1;
     overflow: auto;
     height: 100%;
+
+    @include mid-break {
+      max-height: calc(100% - #{base(4)});
```

Comment:
Instead of restricting height here we should just use bottom padding. This way the scroll container is still full height and feels more natural.

Reply:
There's some css way to achieve this, try a `::after` element with height or vertical padding.